src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +50,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<Person> predicate = (person -> person.getSex().equals(Person.Sex.MAN)
+                && person.getAge() >= fromAge && person.getAge() <= toAge);
```

Comment:
```suggestion
        Predicate<Person> predicate = p -> p.getSex() == (Person.Sex.MAN)
                && p.getAge() >= fromAge && p.getAge() <= toAge;
```
Check it in other methods